docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -210,3 +210,17 @@ The minimum number of join build side rows required to use partitioned join look
 If the build side of a join is estimated to be smaller than the configured threshold,
 single threaded join lookup is used to improve join performance.
 A value of ``0`` disables this optimization.
+
+``optimizer.min-input-rows-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``1000000``
+* **Min allowed value:** ``0``
+* **Session property:** ``min_input_rows_per_task``
+
+The minimum number of rows required per task. This will help optimizer determine hash
+partition count for joins and aggregations. This is majorly used to increase the concurrency
```

Comment:
`This is majorly used to increase the concurrency` -> `Limiting hash partition count for small queries increases concurrency on large clusters where multiple small queries are running concurrently`

Add a comment that it doesn't apply to FTE mode